types/lunr/index.d.ts

```diff
@@ -160,9 +165,13 @@ declare namespace lunr {
          * it should have all fields defined for indexing, though null or undefined values will not
          * cause errors.
          *
+         * Entire documents can be boosted at build time. Applying a boost to a document indicates that
+         * this document should rank higher in search results than other documents.
+         *
          * @param doc - The document to add to the index.
+         * @param attributes - Optional attributes associated with this document.
          */
-        add(doc: object): void;
+        add(doc: object, attributes?: object): void;
```

Comment:
Used `object` due to attributes being set as a value of `_documents` which is just an object.

https://github.com/olivernn/lunr.js/blob/master/lunr.js#L2477
```javascript
this._documents[docRef] = attributes || {}
```
https://github.com/olivernn/lunr.js/blob/master/lunr.js#L2360
```javascript
lunr.Builder = function () {
  this._ref = "id"
  this._fields = Object.create(null)
  this._documents = Object.create(null)
```